src/medical_ui.cpp

```diff
@@ -0,0 +1,1013 @@
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <numeric>
+#include <set>
+#include <string>
+#include <utility>
+
+#include "action.h"
+#include "addiction.h"
+#include "avatar_action.h"
+#include "creature.h"
+#include "character.h"
+#include "character_modifier.h"
+#include "display.h"
+#include "effect.h"
+#include "flag.h"
+#include "game.h"
+#include "output.h"
+#include "ui_manager.h"
+#include "vitamin.h"
+#include "weather.h"
+
+static const efftype_id effect_bite( "bite" );
+static const efftype_id effect_bleed( "bleed" );
+static const efftype_id effect_infected( "infected" );
+
+static const trait_id trait_COLDBLOOD4( "COLDBLOOD4" );
+static const trait_id trait_NOPAIN( "NOPAIN" );
+static const trait_id trait_SUNLIGHT_DEPENDENT( "SUNLIGHT_DEPENDENT" );
+static const trait_id trait_TROGLO( "TROGLO" );
+static const trait_id trait_TROGLO2( "TROGLO2" );
+static const trait_id trait_TROGLO3( "TROGLO3" );
+
+static const flag_id json_flag_SPLINT( "SPLINT" );
+
+enum class medical_tab_mode {
+    TAB_SUMMARY
+};
+
+class selection_line
+{
+    public:
+        selection_line() = default;
+        selection_line( const std::string text, const std::string &desc_str, const int max_width )
+            : desc_str( desc_str ) {
+            std::vector<std::string> textformatted = foldstring( text, max_width,
+                    static_cast<char>( 93 ) );
```

Comment:
Could you explain what is `static_cast<char>( 93 )`?

Reply:
Placed the line separator at ']' to avoid the below consequence, char(93) was corrected by clangtidy to static_cast<char>
>_[_STATUS_]__[_STATUS
>_]_

Reply:
`']'` should do what you want directly.

Reply:
Can you show me what change needs to be done here, if any?